docs/includes/features-lamindb.md

```diff
@@ -32,7 +32,7 @@ Manage storage (files, folders, arrays) and a SQL database (SQLite, Postgres) ba
 
 **Organize and share data across a mesh of LaminDB instances.**
 
-- Create & load database instances like git repos: `lamin init` & `lamin load`
+- Create & load database instances like git repos: [lamin init](cli.md#lamin-init) & [lamin load](cli.md#lamin-load)
```

Comment:
Shall we remove the "database" here in "database instance" given the heading has "lamindb instances" already?

Reply:
In some cases "instances" shoudl be enough, but in most "lamindb instance" should be spelled out. Because nobody knows what a "lamindb instance" is I started to use "database instance" because also nobody understood what "lakehouse instance" is even though that'd be more accurate (that's what we had 2 years ago).

Reply:
Let's remove it then!

Reply:
already done :)